pandas/io/parsers.py

```diff
@@ -37,6 +37,15 @@
 ])
 
 
+class EmptyDataError(ValueError):
```

Comment:
this is essentially an internal error (yes there are specific cases where the user will see it, but not very many)